roqoqo-qryd/tests/integration/api_devices.rs

```diff
@@ -33,7 +33,7 @@ fn test_new_triangular() {
     let device = QrydEmuTriangularDevice::new(Some(1), None, None, None, None);
     let apidevice = QRydAPIDevice::from(&device);
     assert_eq!(device.seed(), 1);
-    assert_eq!(device.seed(), apidevice.seed());
+    assert_eq!(device.seed(), apidevice.seed().unwrap());
```

Comment:
Shouldn't these have the same signature, for consistency?

Reply:
We decided that it's better to have a nullable seed value instead of having a default value of 0 if not specified. So the way APIBackend handles it is correct. It's not necessary to update the old Square and Triangular devices at the moment.

Reply:
Alrighty